example/myapp/tests/test_models.py

```diff
@@ -0,0 +1,21 @@
+from django.test import TestCase
```

Comment:
I'm not sure that this file is necessary -- for TDD it's great to have these, and they'd be great to have if we were developing the example app for use, but I'm not sure that they have much use as a demonstration.

Reply:
When I was first writing the tests I wasn't sure what fields an instance of OAuthConsumer would have, and I thought maybe others would have the same question. That's why I left the model tests. But since we're going to have a documentation, I think we can take it out.

Reply:
nope that's a great answer!